crates/polars-core/src/frame/column/mod.rs

```diff
@@ -657,7 +657,8 @@ impl Column {
                 let mut s = scalar_col.take_materialized_series().rechunk();
                 // SAFETY: We perform a compute_len afterwards.
                 let chunks = unsafe { s.chunks_mut() };
-                chunks[0].with_validity(Some(validity));
```

Comment:
this function isn't in-place and we didn't assign back 😄

Reply:
Auch...